src/stretchy.js

```diff
@@ -9,39 +9,39 @@ const mathMLTree = require("./mathMLTree");
 const utils = require("./utils");
 
 const stretchyCodePoint = {
-    widehat : "^",
-    widetilde : "~",
-    undertilde : "~",
-    overleftarrow : "\u2190",
-    underleftarrow : "\u2190",
-    xleftarrow : "\u2190",
+    widehat: "^",
```

Comment:
I hadn't noticed the difference in `:` position before.  Thanks for aligned this with the style that's used elsewhere.